cmd/chimera/options.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/spf13/viper"
+)
+
+type ServerOptions struct {
+	Addr          string
+	AppsConfig    string `mapstructure:"apps-config"`
```

Comment:
I'm just curious what this `mapstructure` annotation is? I've never seen that before

Reply:
It is for the https://github.com/mitchellh/mapstructure lib. It is used by Viper. Thanks to it we can map options using `Unmarshal` https://github.com/mattermost/chimera/pull/1/files#diff-7ea14ef053eb21da964d22ba8c857fc1be18c314327726c6d326e949b6d5c633R20